source/features/link-to-github-io.tsx

```diff
@@ -30,15 +30,16 @@ function initRepo(signal: AbortSignal): void {
 }
 
 function initRepoList(signal: AbortSignal): void {
-	observe('a[href$=".github.io"][itemprop="name codeRepository"]', addRepoListLink, {signal});
+	// Also consider any old GitHub Pages repo like: resume/resume.github.com , issue: https://github.com/refined-github/refined-github/issues/6228
+	observe(':is(a[href$=".github.io"], a[href$=".github.com"])[itemprop="name codeRepository"]', addRepoListLink, {signal});
```

Comment:
```suggestion
	observe('a:is([href$=".github.io"], [href$=".github.com"])[itemprop="name codeRepository"]', addRepoListLink, {signal});
```

Reply:
I have updated this too. Thanks for reviewing and suggestion 🙂

Reply:
You can "Commit suggestion" in the future as well, if they don't need further changes